### PR TITLE
Updated AL-Go System Files

### PR DESCRIPTION
**Pull request by mazhelez**
No release notes available!

### CODE REVIEW

File: .github/workflows/NextMinor.yaml [hunk 1]
```diff
@@ -125,6 +127,24 @@ jobs:
           settingsJson: ${{ env.Settings }}
           SecretsJson: ${{ env.RepoSecrets }}
 
+      - name: Upload thisbuild artifacts - apps
```
Comment (ankosiorek) [inline .github/workflows/NextMinor.yaml:130]:
```suggestion
      - name: Upload this build artifacts - apps
```

File: .github/workflows/PullRequestHandler.yaml [hunk 2]
```diff
@@ -41,11 +41,13 @@ jobs:
           Add-Content -Path $env:GITHUB_OUTPUT -Value "prfolder=$prfolder"
           $prPath = Join-Path $location $prFolder
           New-Item -Path $prPath -ItemType Directory | Out-Null
+          $prfilesChanged = @()
```
Comment (ankosiorek) [inline .github/workflows/PullRequestHandler.yaml:44]:
```suggestion
          $prFilesChanged = @()
```

File: .github/workflows/NextMajor.yaml [hunk 3]
```diff
@@ -125,6 +127,24 @@ jobs:
           settingsJson: ${{ env.Settings }}
           SecretsJson: ${{ env.RepoSecrets }}
 
+      - name: Upload thisbuild artifacts - apps
```
Comment (ankosiorek) [inline .github/workflows/NextMajor.yaml:130]:
```suggestion
      - name: Upload build artifacts - apps
```

File: .github/workflows/NextMinor.yaml (hunk 1 above)
Comment (ankosiorek) [reply .github/workflows/NextMinor.yaml:130]: Is that some kind of convention I don't know: _thisbuild_?

Comment (mazhelez) [reply .github/workflows/NextMinor.yaml:130]: Yeah, this comes from AL-Go. It was updated in [this PR](https://github.com/microsoft/AL-Go/pull/297/files#diff-01f4ef19a96cdb7663a0199b5880386ffc66ae75cf8696fbc655bcc2eb5cc316).

File: .github/workflows/PullRequestHandler.yaml (hunk 2 above)
Comment (mazhelez) [reply .github/workflows/PullRequestHandler.yaml:44]: Again, this needs to be fixed in AL-Go itself :)

**Review by freddydk — APPROVED** [review]

File: .github/workflows/NextMinor.yaml (hunk 1 above)
Comment (freddydk) [reply .github/workflows/NextMinor.yaml:130]: The thisbuild artifacts are artifacts from this build, which only is used for this build to carry over to the next project (when running multiprojects with useProjectDependencies = true) - a fix for this issue: https://github.com/microsoft/AL-Go/issues/292

File: .github/workflows/PullRequestHandler.yaml (hunk 2 above)
Comment (freddydk) [reply .github/workflows/PullRequestHandler.yaml:44]: Yeah - we can fix this in AL-Go

**Pull request merged**